routes/lesson.js

```diff
@@ -1,5 +1,6 @@
 const router = require("express").Router();
 const lesson = require("../model/lesson");
+const chapter = require('../models/chapter');
```

Comment:
You shouldn't be importing the script here. Remember, the script and the backend are separate.